### PR TITLE
Add SB and SBT enums in Interop ComCtl32 and SBS and SS enums in User32

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Add SB and SBT enums in Interop ComCtl32 and SBS and SS enums in User32. 
- Remove SB, SBT, SBS and SS constants from NativeMethods.cs and replace their usages with the above enums.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2588)

### CODE REVIEW

File: src/Common/src/Interop/ComCtl32/Interop.SB.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class ComCtl32
+    {
+        [Flags]
+        public enum SB : uint
+        {
+            GETTEXT = User32.WM_USER + 13,
```
Comment (RussKie) [inline src/Common/src/Interop/ComCtl32/Interop.SB.cs]: * Import `Interop.User32` to simplify the definitions
* Order in the value asc

File: src/Common/src/Interop/ComCtl32/Interop.SBT.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class ComCtl32
+    {
+        [Flags]
+        public enum SBT : uint
+        {
+            OWNERDRAW = 0x1000,
```
Comment (RussKie) [inline src/Common/src/Interop/ComCtl32/Interop.SBT.cs]: Order in value asc

File: src/Common/src/Interop/User32/Interop.SBS.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [Flags]
+        public enum SBS : uint
+        {
+            HORZ = 0x0000,
```
Comment (RussKie) [inline src/Common/src/Interop/User32/Interop.SBS.cs]: ditto

File: src/System.Windows.Forms/src/System/Windows/Forms/StatusBarPanel.cs [hunk 4]
```diff
@@ -619,30 +619,30 @@ internal void Realize()
                 switch (borderStyle)
                 {
                     case StatusBarPanelBorderStyle.None:
-                        border |= NativeMethods.SBT_NOBORDERS;
+                        border |= (int)ComCtl32.SBT.NOBORDERS;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/StatusBarPanel.cs]: `border` should be declared as `SBT border`

File: src/System.Windows.Forms/src/System/Windows/Forms/StatusBarPanel.cs [hunk 5]
```diff
@@ -577,7 +577,7 @@ internal void Realize()
             {
                 string text;
                 string sendText;
-                int border = 0;
+                ComCtl32.SBT border = 0;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/StatusBarPanel.cs]: Import `Interop.ComCtl32` to simplify the definitions

**Comment by RussKie** [comment]
Sorry @gpetrou there are MC again

**Comment by gpetrou** [comment]
@RussKie you can merge this now.

**Pull request merged**